cmake/machinecfg/Darwin.cmake

```diff
@@ -0,0 +1,157 @@
+#========================================================================================
+# Parthenon performance portable AMR framework
+# Copyright(C) 2020 The Parthenon collaboration
+# Licensed under the 3-clause BSD License, see LICENSE file for details
+#========================================================================================
+# (C) (or copyright) 2020. Triad National Security, LLC. All rights reserved.
+#
+# This program was produced under U.S. Government contract 89233218CNA000001 for Los
+# Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+# for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+# in the program are reserved by Triad National Security, LLC, and the U.S. Department
+# of Energy/National Nuclear Security Administration. The Government is granted for
+# itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+# license in this material to reproduce, prepare derivative works, distribute copies to
+# the public, perform publicly and display publicly, and to permit others to do so.
+#========================================================================================
+
+# OPTIONS:
+# - `DARWIN_ARCH` - define which processor is being used
+#       Default:
+#           `DARWIN_ARCH` environment variable, if it exists, otherwise
+#           the result of `uname -m`.
+#       Possible Values:
+#           `x86_64` (assumes at least haswell)
+#           `ppc64le` (assumes power9 + volta gpus)
+# - `DARWIN_VIEW_DATE` - The date the dependencies were installed.
+#       Default:
+#           unset, which results in using the view associated with your
+#           current commit. See `DARWIN_VIEW_DATE_LATEST`
+# - `DARWIN_COMPILER` - Compiler family to use
+#       Default: "GCC"
+#       Possible Values: "GCC", "GCC9"
+
```

Comment:
It might be nice to add the ${MACHINE_VARIANT} variable that @pgrete makes use of in the config files he has. To indicate what resources are actually being used.

Reply:
Ah, yeah, currently it always builds MPI. hmmm... this will take a little while, but I'll do this.

Reply:
We decided offline to add this later when we need it.